From dee452cc48cf8a9dbca172600107c2f60bec3d3f Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Tue, 29 Sep 2020 10:13:33 +0200
Subject: [PATCH] sunxi: dts: OrangePi Zero: Add SPI aliases to make bus usable
 with u-boot.

The u-boot code relies on aliases to assign bus number.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm/dts/sun8i-h2-plus-orangepi-zero.dts | 1 +
 arch/arm/dts/sunxi-spi-u-boot.dtsi           | 8 ++++++++
 2 files changed, 9 insertions(+)
 create mode 100644 arch/arm/dts/sunxi-spi-u-boot.dtsi

diff --git a/arch/arm/dts/sun8i-h2-plus-orangepi-zero.dts b/arch/arm/dts/sun8i-h2-plus-orangepi-zero.dts
index 3706216ffb..c3660f72d9 100644
--- a/arch/arm/dts/sun8i-h2-plus-orangepi-zero.dts
+++ b/arch/arm/dts/sun8i-h2-plus-orangepi-zero.dts
@@ -46,6 +46,7 @@
 /dts-v1/;
 #include "sun8i-h3.dtsi"
 #include "sunxi-common-regulators.dtsi"
+#include "sunxi-spi-u-boot.dtsi"
 
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
diff --git a/arch/arm/dts/sunxi-spi-u-boot.dtsi b/arch/arm/dts/sunxi-spi-u-boot.dtsi
new file mode 100644
index 0000000000..df89d02ff2
--- /dev/null
+++ b/arch/arm/dts/sunxi-spi-u-boot.dtsi
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0+
+/ {
+
+	aliases {
+		spi0 = &spi0;
+		spi1 = &spi1;
+	};
+};
